### ISSUE TITLE
Provide visual code default code formatting settings 

### ISSUE THREAD

**Issue by hariprasadms** [invertase/zapp.run#42]
**Is your feature request related to a problem? Please describe.**
There is no way to formate the code lines equivalent VC code, code formate settings. Providing an option on editor like a button or settings where users can formate the code lines, will be more useful.

**Describe the solution you'd like**

**Additional context**
Basically user should be able to format code using shortcuts or a button pressing on the editor.

**Comment by hariprasadms** [comment]
I can see that On Mac Shift + Option + F shortcut works fine. However I button on editor UI would be easy and quick access to the users. Thanks

**Comment by Salakar** [comment]
It's also available in the Command Palette (Cmd + P on mac) - would you still require a button for this?

**Comment by hariprasadms** [comment]
Yes. a Code formate button on editor UI will be more user-friendly. I personally definitely prefer one on UI please :)

**Comment by Ehesp** [comment]
Is there a visual for this in VSCode? Conscious of cluttering the UI with lots of individual buttons for things.

**Comment by blaugold** [comment]
This is where StackBlitz has a button for formatting:

![Screenshot 2022-07-15 at 14 49 01](https://user-images.githubusercontent.com/5929595/179226456-281e196f-8f3d-41ea-98d7-cefb8aff3787.png)

**Comment by Ehesp** [comment]
I currently have support for actions for files; e.g.

![image](https://user-images.githubusercontent.com/842078/179230031-4bd8221a-e091-49ba-b1cb-5f2dd82887c6.png)

I guess the problem here is that a format won't be triggered until analyzer is ready... cc @Salakar for thoughts

**Comment by hariprasadms** [comment]
> This is where StackBlitz has a button for formatting:
> 
> ![Screenshot 2022-07-15 at 14 49 01](https://user-images.githubusercontent.com/5929595/179226456-281e196f-8f3d-41ea-98d7-cefb8aff3787.png)

yes this place UI looks like a perfect place for button. I agree this one.

**Comment by novas1r1** [comment]
I think it would be fine to maybe put it into the shortcuts list? I would not use a button for that, but didn't know about the shortcut.
